packages/go/openapi/src/schemas/model.user.yaml

```
@@ -65,3 +65,8 @@ allOf:
       all_environments:
         type: boolean
         readOnly: true
+      environment_access_control:
+        description: A collection of environments that a user has been granted access to as part of ETAC
+        type: array
+        items:
+          $ref: './model.environment-access-control.yaml'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)